developer/src/kmc-kmn/src/ng-compiler/token-buffer.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Keyman is copyright (C) SIL Global. MIT License.
+ *
+ * Created by Dr Mark C. Sinclair on 2025-03-13
+ *
+ * KMC KMN Next Generation Parser (TokenBuffer)
+ */
+
+import { Token, TokenTypes } from "./lexer.js";
+
+export class TokenBuffer {
+  private list: Token[];
+  private _cp: number;
```

Comment:
```suggestion
  private _currentPosition: number;
```

unless it is going to be invoked many other places?

Reply:
Okay ... better for readability